### ISSUE TITLE
No dynamic OPML supported

### ISSUE THREAD

**Issue by tobiasholle** [Ranchero-Software/NetNewsWire#4372]
Is it possible to support dynamic OPML to import Folders and Feeds i.e. through FreshRSS?

Best, Tobias

**Comment by brentsimmons** [comment]
Does FreshRSS support dynamic OPML?

**Comment by tobiasholle** [comment]
Yes: https://github.com/FreshRSS/FreshRSS/discussions/4769#discussioncomment-10810382

**Comment by tobiasholle** [comment]
Is there any plans to include dynamic OPML in NetNewsWire? It would be really helpful for sharing feeds that automatically update.

**Comment by hbprotoss** [comment]
> Is there any plans to include dynamic OPML in NetNewsWire? It would be really helpful for sharing feeds that automatically update.

+1

**Comment by brentsimmons** [comment]
Duplicate of #626
